src/installer/features/profiles/config/_mixins/_docs-paid/skills/paid-write-noridoc/SKILL.md

```diff
@@ -23,26 +23,56 @@ Use regular Write tool instead for:
 ## Usage
 
 ```bash
-node ~/.claude/skills/write-noridoc/script.js --filePath="@/server/src/persistence" --content="# Documentation" [--gitRepoUrl="https://github.com/..."]
+node ~/.claude/skills/write-noridoc/script.js --filePath="@<repository>/path" --content="# Documentation" [--gitRepoUrl="https://github.com/..."]
 ```
 
 ## Parameters
 
-- `--filePath` (required): Path where doc would normally live (e.g., "@/server/src/persistence")
+- `--filePath` (required): Path in format `@<repository>/<path>` (e.g., `@nori-watchtower/server/src/persistence`)
+  - Use `@<repository>/path` for repository-scoped docs
+  - Use `@/path` or plain paths for `no-repository` scope (backward compatible)
 - `--content` (required): Markdown content
 - `--gitRepoUrl` (optional): Link to git repository
 
-## Example
+## Repository Detection
+
+The repository is automatically extracted from the filePath by the server:
+
+- `@nori-watchtower/server/src/api` → repository: `nori-watchtower`
+- `@/server/src/api` → repository: `no-repository`
+- `server/src/api` → repository: `no-repository`
+
+Repository names must be **lowercase letters, numbers, and hyphens** only.
+
+## Examples
+
+### Create repository-scoped noridoc
 
 ```bash
 node ~/.claude/skills/write-noridoc/script.js \
-  --filePath="@/plugin/src/api" \
+  --filePath="@nori-watchtower/server/src/api" \
   --content="# API Client
 
 Provides access to Nori backend." \
-  --gitRepoUrl="https://github.com/theahura/nori-agent-brain"
+  --gitRepoUrl="https://github.com/username/nori-watchtower"
```

Comment:
replace the nori-watchtower naming with a non-specific repo name (here and throughout)